test-foundry/TestStessHeapOrdering.t.sol

```diff
@@ -14,9 +14,9 @@ contract HeapStorage {
     function setUp() public {
         for (uint256 i = 0; i < TESTED_SIZE; i++) {
```

Comment:
```suggestion
        for (uint96 i; i < TESTED_SIZE; i++) {
```

Reply:
as well for the `TESTED_SIZE`

Reply:
`uint96` is (mostly) not needed outside of the library now, because the casting is done inside